src/translations/translations.fr.js

```diff
@@ -515,6 +515,71 @@ export const translations = {
+  'cc-orga-member-list.error': `Une erreur est survenue pendant le chargement de la liste des membres.`,
+  'cc-orga-member-list.error-member-not-found.heading': `Membre introuvable`,
+  'cc-orga-member-list.error-member-not-found.text': () => sanitize`<p>Le membre a probablement quitté l'organisation ou a été supprimé par quelqu'un d'autre pendant que vous consultiez la liste.<p><p>Veuillez <strong>rafraîchir votre page</strong> pour récupérer la liste des membres à jour.</p>`,
+  'cc-orga-member-list.error.unauthorised.heading': `Vous n'avez pas les droits nécessaires`,
+  'cc-orga-member-list.error.unauthorised.text': `Seul un admin peut inviter, éditer ou supprimer un autre admin.`,
+  'cc-orga-member-list.filter.mfa': `Comptes non sécurisés par 2FA`,
+  'cc-orga-member-list.filter.name': `Filtrer par nom ou adresse e-mail`,
+  'cc-orga-member-list.invite.email.error-duplicate': `Cet utilisateur fait déjà partie des membres de votre organisation.`,
+  'cc-orga-member-list.invite.email.error-empty': `Veuillez saisir une adresse e-mail.`,
+  'cc-orga-member-list.invite.email.error-format': () => sanitize`Format d'adresse e-mail invalide.<br>Exemple: john.doe@example.com.`,
+  'cc-orga-member-list.invite.email.format': `nom@example.com`,
+  'cc-orga-member-list.invite.email.label': `Adresse e-mail`,
+  'cc-orga-member-list.invite.heading': `Inviter un membre`,
+  'cc-orga-member-list.invite.info': () => sanitize`Plus d'informations à propos des rôles sur la page <a href="https://www.clever-cloud.com/doc/account/organizations/#roles-and-privileges">Rôles et organisations (en anglais)</a>`,
+  'cc-orga-member-list.invite.role.accounting': `Comptable`,
+  'cc-orga-member-list.invite.role.admin': `Admin`,
+  'cc-orga-member-list.invite.role.developer': `Développeur`,
+  'cc-orga-member-list.invite.role.label': `Rôle`,
+  'cc-orga-member-list.invite.role.manager': `Manager`,
+  'cc-orga-member-list.invite.submit': `Inviter`,
+  'cc-orga-member-list.invite.submit.error': ({ userEmail }) => sanitize`Une erreur est survenue lors de l'invitation de <strong>${userEmail}</strong> dans l'organisation.`,
+  'cc-orga-member-list.invite.submit.error-rate-limit.message': `Attendez quelques minutes avant d'essayer à nouveau.`,
+  'cc-orga-member-list.invite.submit.error-rate-limit.title': `Vous avez tenté d'inviter des membres trop de fois`,
+  'cc-orga-member-list.invite.submit.success': ({ userEmail }) => sanitize`Un e-mail a été envoyé à <strong>${userEmail}</strong> pour l'inviter dans l'organisation.`,
+  'cc-orga-member-list.leave.btn': `Quitter l'organisation`,
+  'cc-orga-member-list.leave.error': `Une erreur est survenue lorsque vous avez tenté de quitter l'organisation.`,
+  'cc-orga-member-list.leave.error-last-admin.heading': `Vous êtes le dernier admin de l'organisation`,
+  'cc-orga-member-list.leave.error-last-admin.text': `Veuillez designer un nouvel admin avant de pouvoir quitter l'organisation.`,
```

Comment:
Misses an accent on "désigner"

```suggestion
  'cc-orga-member-list.leave.error-last-admin.text': `Veuillez désigner un nouvel admin avant de pouvoir quitter l'organisation.`,
```